zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -524,6 +521,7 @@ def convert_slack_workspace_messages(slack_data_dir: str, users: List[ZerverFiel
 
     logging.info('######### IMPORTING MESSAGES STARTED #########\n')
 
+    message_json = {}
```

Comment:
If you're shifting this later, can you construct the dict in-place further down with `{"a": b, "c": d}` or `dict(a=b, c=d)` form?